src/Casdoor.AspNetCore/CasdoorIdentity.cs

```diff
@@ -0,0 +1,224 @@
+using Microsoft.AspNetCore.Identity;
+using System.Threading.Tasks;
+using System.Threading;
+using Casdoor.Client;
+
+namespace Casdoor.AspNetCore.Authentication
+{
+    public class CasdoorIdentityUserStore : IUserStore<CasdoorUser>
+    {
+        private CasdoorClient _casdoorClient;
+        public CasdoorIdentityUserStore(CasdoorClient casdoorClient)
+        {
+            _casdoorClient = casdoorClient;
+        }
+        //
+        // 摘要:
```

Comment:
You are encouraged to add comments to code, this is a very good thing. But we should add all English comments. And it would be better if the comment format follows some kind of standard or convention in DotNet world. Plz change all.